cmd/disk-cache-format.go

```diff
@@ -0,0 +1,440 @@
+/*
```

Comment:
you can rename this file to format-disk-cache.go similar to format-fs.go and format-xl.go